vignettes/ebola.Rmd

```diff
@@ -0,0 +1,620 @@
+  add_linpred_draws(fit_sex_district, dpar = TRUE)
+```
+
+(ref:linpred-sex-district) The posterior distribution of the linear predictor of `mu` parameter within the sex-district stratified model.
+
+```{r linpred-sex-district, fig.cap="(ref:linpred-sex-district)"}
+linpred_draws_sex_district |>
+  mutate(
+    sex = case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = mu, y = district)) +
+  stat_pointinterval() +
+  facet_grid(. ~ sex) +
+  labs(x = "Posterior of the mu linear predictor", y = "") +
+  theme_minimal()
+```
+
+## Delay probability mass functions
+
+Posterior predictions of the delay distribution are an important output of an analysis with the `epidist` package.
+In this section, we demonstrate how to produce either a discrete probability mass function representation, or continuous probability density function representation of the delay distribution.
+
+### Discrete probability mass function
+
+The posterior predictive distribution under no truncation but daily censoring.
+
+No stratification.
+
+```{r}
+newdata_pmf <- obs_prep |>
+  as.data.frame() |>
+  mutate(obs_t = 1000, pwindow_upr = 1, swindow_upr = 1)
+
+draws_pmf <- predicted_draws(fit, newdata = newdata_pmf, ndraws = 1000)
+```
+
+There were `r round(mean(draws_pmf$.prediction > 30) * 100, 1)`% of delays greater than 30, which are omitted in the following figure.
+
+```{r}
+ggplot(draws_pmf, aes(x = .prediction)) +
+  geom_bar(aes(y = after_stat(count / sum(count)))) +
+  labs(x = "PMF with daily censoring and no truncation", y = "") +
+  scale_x_continuous(limits = c(0, 30)) +
+  theme_minimal()
+```
+
+Stratification by sex:
+
+```{r}
+newdata_sex_pmf <- obs_prep |>
+  as.data.frame() |>
+  data_grid(sex) |>
+  mutate(obs_t = 1000, pwindow_upr = 1, swindow_upr = 1)
+
+draws_sex_pmf <- predicted_draws(
+  fit_sex,
+  newdata = newdata_sex_pmf,
+  ndraws = 1000
+)
+```
+
+```{r}
+draws_sex_pmf |>
+  mutate(
+    sex = case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .prediction)) +
+  geom_bar(aes(y = after_stat(count / sum(count)))) +
+  labs(x = "PMF with daily censoring and no truncation", y = "") +
+  facet_grid(. ~ sex) +
+  theme_minimal()
+```
+
+Stratification by sex-district:
+
+```{r}
+newdata_sex_district_pmf <- obs_prep |>
+  as.data.frame() |>
+  data_grid(sex, district) |>
+  mutate(obs_t = 1000, pwindow_upr = 1, swindow_upr = 1)
+
+draws_sex_district_pmf <- predicted_draws(
+  fit_sex_district,
+  newdata = newdata_sex_district_pmf,
+  ndraws = 1000
+)
+```
+
+```{r}
+draws_sex_district_pmf |>
+  mutate(
+    sex = case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .prediction)) +
+  geom_bar(aes(y = after_stat(count / sum(count)))) +
+  labs(x = "PMF with daily censoring and no truncation", y = "") +
+  facet_grid(district ~ sex) +
+  theme_minimal()
```

Comment:
This plot is barely legible. You could make it much taller or consider overplotting with colors by district?

Reply:
I think it should be more legible now using the HMC version.